src/libs/campaign/useCampaignTracker.ts

```diff
@@ -5,18 +5,27 @@ import { analytics } from 'libs/analytics'
 import { env } from 'libs/environment'
 import { useTrackingConsent } from 'libs/trackingConsent'
 
+// We ask for the user's consent on app launch. We defer the sending of events to AppsFlyer,
+// depending on the user's response, by the following delay.
+// Reference:
+// https://support.appsflyer.com/hc/en-us/articles/207032066-iOS-SDK-V6-X-integration-guide-for-developers#integration-33-configuring-app-tracking-transparency-att-support
+const TIME_TO_WAIT_FOR_ATT_CONSENT = 60 // in seconds
+
 export const useCampaignTracker = () => {
-  const consentTracking = useTrackingConsent()
+  // First we ask for the user's consent to access their IDFA information
+  useTrackingConsent()
 
   useEffect(() => {
-    if (__DEV__ || !consentTracking) return
+    if (__DEV__) return
 
```

Comment:
❓ du coup comment on prend en compte la réponse de l'utiliasteur ?

Reply:
Avec `timeToWaitForATTUserAuthorization`. Si il coonsent, on peut récupérer son IDFA, sinon non.

Doc de configuration AppsFlyer: https://support.appsflyer.com/hc/en-us/articles/207032066-iOS-SDK-V6-X-integration-guide-for-developers#integration-33-configuring-app-tracking-transparency-att-support